lxd/api10.go

```diff
@@ -1,17 +1,12 @@
 package main
 
 import (
-	"crypto/rand"
-	"encoding/hex"
 	"fmt"
-	"io"
 	"net/http"
 	"syscall"
 
-	"golang.org/x/crypto/scrypt"
-	"gopkg.in/lxc/go-lxc.v2"
-
 	"github.com/lxc/lxd/shared"
+	"gopkg.in/lxc/go-lxc.v2"
```

Comment:
If I read the above diff right, you're listing go-lxc in the same import section as lxd/shared, that doesn't match the coding style for the rest of the project.

Imports in LXD should be:
 - Built-ins
 - External
 - LXD

Every block should be separated from the other with a blank line and then sorted (as enforced by go fmt).